docs/website/mkdocs.yml

```
@@ -112,6 +112,7 @@ nav:
       - "building-from-source/python-bindings-and-importers.md"
       - "building-from-source/android.md"
       - "building-from-source/riscv.md"
+      - "building-from-source/ios.md"
```

nit: I'd put iOS after Android (group mobile phone operating systems together, and alphabetical)

```suggestion
      - "building-from-source/android.md"
      - "building-from-source/ios.md"
      - "building-from-source/riscv.md"
```